cpp/src/slice2cpp/Gen.cpp

```diff
@@ -1820,11 +1858,22 @@ Slice::Gen::ProxyVisitor::visitOperation(const OperationPtr& p)
     H << sp;
     if (comment)
     {
-        StringList postParams, returns;
-        postParams.push_back("@param " + responseParam + " The response callback.");
+        StringList postParams;
+        if (lambdaOutParams.size() > 0)
```

Comment:
Also why not this:
```suggestion
        if (!lambdaOutParams.empty())
```
Although if you do this, I would swap the if/else so could avoid checking a negation.